py/qqa/plots/plotimage.py

```diff
@@ -89,7 +89,8 @@ def main(input_in = None, output_in = None, downsample_in = None):
         n = downsample_in
         output = output_in
 
-    image = fitsio.read(image)
+    hdul = fits.open(image)
+    image = hdul[0].data
```

Comment:
This might leave the file open even after the function has returned (astropy.io.fits does some bad stuff with caching global state and memory mapping).  Better to either
```python
image = fits.getdata(filename, 0)
```
or
```python
with fits.open(filename) as hdul:
    image = hdul[0].data
```

Also: this pre-dates this PR, but while we're at it: it is potentially confusing style to have `image` start out as a filename and then switch to being the data itself.  Let's keep variable names clear with `input_in` or `filename` or similar, and not change the meaning of a variable mid-function.